MeqNodes/src/FFTBrick.cc

```diff
@@ -244,7 +244,7 @@ inline void copyDoubleToComplex (
   blitz::Array<dcomplex,2>::iterator iter_out = out1.begin();
   blitz::Array<double,2>::iterator iter_in = in1.begin();
   for( ; iter_out != out1.end(); ++iter_out,++iter_in)
-    *iter_out = *iter_in + 0.i;
+    *iter_out = *iter_in + 0;
```

Comment:
Why this change to float point? This should implicitly cast to complex

Reply:
This relates to https://github.com/ska-sa/meqtrees-timba/commit/49b1f1b1c2c2d40c562b83d940f44ef4db17a3ca

When i reverted my accidental changes I reverted this changes too. So here I'm trying to take things back to how there where